services/repository/contributors_graph_test.go

```diff
@@ -18,6 +19,8 @@ import (
 
 func TestRepository_ContributorsGraph(t *testing.T) {
 	assert.NoError(t, unittest.PrepareTestDatabase())
+	system_model.SetSettings(db.DefaultContext, map[string]string{setting.Config().Picture.DisableGravatar.DynKey(): "defer"})
```

Comment:
As above, we could modify the test case to use a local avatar link.

The gravatar could be (should have been) tested separately, but not in other non-related tests.

Reply:
Implemented here: https://github.com/go-gitea/gitea/pull/32529/commits/b05e55c2abe3685e90e410e75ae72f4d50be7931

I also verified that the existing gravatar implementation is tested in https://github.com/go-gitea/gitea/blob/main/models/avatars/avatar_test.go#L44, so this change doesn't reduce test coverage.